src/config/helmet.ts

```diff
@@ -95,7 +99,12 @@ export const webchatHelmetConfiguration: HelmetOptions = {
         "https://*.smartagent.app",
         "https://*.ruxit.com",
         "https://*.dynatrace.com",
+        "https://participant.connect.eu-west-2.amazonaws.com",
+        (req: Request, res: Response): string =>
+          `'${res.locals.missionLabWebSocketAddress}'`,
```

Comment:
why does this need to be retreived from the response local object? Can it just be taken from `config.ts` ?

Reply:
I did this way to align with how it was done previously. I didn't see any example where we were retrieving from config.ts